src/content/docs/sandbox-management.mdx

```diff
@@ -172,9 +184,82 @@ console.log(sandbox.state)
 
 To get the preview URL for a specific port, check out [Preview & Authentication](/docs/preview-and-authentication).
 
-## Stop & Remove Sandbox
+## Stop and Resume Sandbox
+
+The Daytona SDK provides methods to stop and resume Sandboxes using Python and TypeScript.
```

Comment:
Use `start` instead of `resume`.

We have plans on implementing actual `pause` and `resume` methods that will pause/resume the memory state as well so we don't want that to conflict with `start`.